corehq/apps/geospatial/management/commands/index_geolocation_case_properties.py

```diff
@@ -0,0 +1,86 @@
+import math
+
+from django.core.management.base import BaseCommand
+
+from dimagi.utils.chunked import chunked
+
+from corehq.apps.es import CaseSearchES, case_search_adapter, filters, queries
+from corehq.apps.es.case_search import (
+    CASE_PROPERTIES_PATH,
+    PROPERTY_GEOPOINT_VALUE,
+    PROPERTY_KEY,
+)
+from corehq.apps.es.client import manager
+from corehq.apps.geospatial.utils import get_geo_case_property
+from corehq.form_processor.models import CommCareCase
+from corehq.util.log import with_progress_bar
+
+DEFAULT_QUERY_LIMIT = 10_000
+DEFAULT_CHUNK_SIZE = 100
+
+
+class Command(BaseCommand):
+    help = 'Index geopoint data in ES'
+
+    def add_arguments(self, parser):
+        parser.add_argument('domain')
+        parser.add_argument('--case_type', required=False)
+        parser.add_argument('--query_limit', required=False)
+        parser.add_argument('--chunk_size', required=False)
+
+    def handle(self, *args, **options):
+        domain = options['domain']
+        case_type = options.get('case_type')
+        query_limit = options.get('query_limit')
+        chunk_size = options.get('chunk_size')
+        self.index_case_docs(domain, query_limit, chunk_size, case_type)
+
+
+def index_case_docs(domain, query_limit=DEFAULT_QUERY_LIMIT, chunk_size=DEFAULT_CHUNK_SIZE, case_type=None):
+    geo_case_property = get_geo_case_property(domain)
+    query = _case_query(domain, geo_case_property, case_type)
+    count = query.count()
+    print(f'{count} case(s) to process')
+    batch_count = math.ceil(count / query_limit)
+    print(f"Cases will be processed in {batch_count} batches")
+    for i in range(0, batch_count):
```

Comment:
nit:

```suggestion
    for i in range(batch_count):
```

Reply:
Addressed in [807e5c8](https://github.com/dimagi/commcare-hq/pull/35090/commits/807e5c8c16605a3f625f5c82cc998d0588e55b1d).